lib/errors.ts

```diff
@@ -140,7 +140,7 @@ const EXPECTED_ERROR_REGEXES = [
 	/^BalenaDeviceNotFound/, // balena-sdk
 	/^BalenaExpiredToken/, // balena-sdk
 	/^Missing \w+$/, // Capitano,
-	/^Missing \d required argument/, // oclif parser: RequiredArgsError, RequiredFlagError
+	/^Missing \d+ required arg/, // oclif parser: RequiredArgsError, RequiredFlagError
```

Comment:
This change is not necessary if the CLI installs correctly, but it is still useful in case the user runs `npm install balena-cli` as the root user without adding the `--unsafe-perm` option. In this case, the `postinstall` npm script is skipped, oclif is not patched and the error message reverts to oclif's original `Missing 1 required args:`. While testing on Linux as the root user, I ran `npm install && npm test` on the `balena-cli` dir and the result was failed tests (which is OK: broken install) and error reports sent to Sentry. This code change avoids such Sentry reporting.